Sources/xcproj/PBXGroup.swift

```diff
@@ -39,13 +37,12 @@ final public class PBXGroup: PBXFileElement {
         self.usesTabs = usesTabs
         self.indentWidth = indentWidth
         self.tabWidth = tabWidth
-        super.init(reference: reference, sourceTree: sourceTree, path: path, name: name)
+        super.init(sourceTree: sourceTree, path: path, name: name)
     }
 
     public static func == (lhs: PBXGroup,
```

Comment:
Just wondering: Is there any reason why your not using AutoEquatable from Sourcery?

Reply:
Not really. I actually think it's a good idea to automate the generation of init and the conformance of Equatable. Creating an issue 😛

Reply:
https://github.com/xcodeswift/xcproj/issues/186